src/firefly/js/ui/SuggestBoxInputField.jsx

```diff
@@ -260,12 +260,12 @@ export class SuggestBoxInputFieldView extends PureComponent {
     render() {
 
         const {displayValue, valid, message, highlightedIdx, isOpen, inputWidth, suggestions, mouseTrigger } = this.state;
-        const {label, tooltip, inline, renderSuggestion, wrapperStyle, placeholder, slotProps={}, sx,
+        const {label, tooltip, renderSuggestion, wrapperStyle, placeholder, slotProps={}, sx,
```

Comment:
do we still `wrapperStyle`, can you take it out?

Reply:
I didn't touch SuggestBoxInputField on purpose because besides these old style props, the suggestion box also needs to be changed and it's used in multiple places so I'd need to test thoroughly. I have it mentioned on outstanding issues ticket - feel free to create a separate ticket for it and assign it to me.

Reply:
ok, we can put that off then